content/webapp/pages/api/series/[id].ts

```diff
@@ -1,27 +0,0 @@
-import { isString } from '@weco/common/utils/array';
```

Comment:
do we not use this anywhere?

Reply:
Not that I can find; gripping the codebase for `api/series` where we'd be using this endpoint doesn't turn up anything.